chatbot.py

```diff
@@ -8,30 +8,32 @@
 from utilities import learn_question_response
 from utilities import add_learnt_statement_to_database
 from utilities import setup_nltk
-from googleMapsApiModule import direction
-from googleMapsApiModule import add_to_maps_database
-from googleMapsApiModule import get_from_maps_database
-from googleMapsApiModule import geocoding
-
+import enum
+class symbolic_constant(enum.Enum): 
+    msg=0
+    train_me=1
+    ogn=2
+    desti=3
+    
 def setup():
     setup_nltk()
     clf = classify_model()
     setup_database()
-    learn_response = 0
+    learn_response = 'msg'
```

Comment:
I think enum keys are mostly used like this `symbolic_constant.msg.name` rather than `"msg"`

Reply:
With the above change, it'd rather look something like `LearnResponse.MESSAGE.name`. Please reflect such changes in all instances within the PR 😄